templates/CRM/Admin/Form/Options.tpl

```diff
@@ -38,22 +38,22 @@
                 <span class="description">{ts}A "string value" or regular expression to be redacted (replaced).{/ts}</span>
              </td>
            </tr>
-        {else}
+        {elseif !empty($form.label)}
```

Comment:
This would also hide financial_account_id when it didn't before, since despite the indenting fin id is within this block. But I'm not sure what screen that comes up on. Clicking around the related screens looks ok.